docs/python/programming/math/vector.rst

```diff
@@ -549,6 +549,33 @@ Vector
     .. seealso::
         :meth:`Vector.record`, :meth:`Vector.play_remove`
     
+    Example of playing into an Iclamp for varying current:
+
+        .. code-block::
+            python
+        
+        from neuron import h
```

Comment:
The Python code should be indented to align with the word "python" above. Personally, I'd un indent the above, but I see that wasn't done in the next bit.

Reply:
all aligned now